test/BasicNetworkTest.template.ts

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2021 SYMBOL
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect } from 'chai';
+import { Nis1NetworkType, SymbolNetworkType } from '../src/core/constants';
+
+type Network = SymbolNetworkType | Nis1NetworkType;
+
```

Comment:
please have test with determininistic mapping of public key => (a) PUBLIC address, (b) PUBLIC_TEST address 

[see sdk-python for example]

Reply:
To create an Address actually is on Address class rather than Network class itself.